src/main/java/com/tech/H2Databaseinfo/controller/Home.java

```diff
@@ -0,0 +1,5 @@
+package com.tech.H2Databaseinfo.controller;
+
```

Comment:
Can you place annotate the class with the proper  Annotation. 
and 
create method for testing the return at least "Hello World"